### PR TITLE
Add psea assessment fields to partner

### PR DESCRIPTION
**Pull request by reinbach**
[ch19532]
[ch18002]

### CODE REVIEW

File: src/etools/applications/partners/models.py [hunk 1]
```diff
@@ -490,6 +490,29 @@ class PartnerOrganization(TimeStampedModel):
...
+    )
+    sea_risk_rating_name = models.CharField(
+        max_length=150,
+        verbose_name=_("PSEA Risk Rating"),
+        blank=True,
+        default='',
+    )
+    highest_risk_rating_type = models.CharField(
+        max_length=150,
+        verbose_name=_("HACT Risk Rating Type"),
```
Comment (robertavram) [inline src/etools/applications/partners/models.py:506]: change this to "Highest Risk Rating Type" not "HACT"

File: src/etools/applications/partners/models.py [hunk 2]
```diff
@@ -490,6 +490,29 @@ class PartnerOrganization(TimeStampedModel):
...
+        blank=True,
+        default='',
+    )
+    highest_risk_rating_name = models.CharField(
+        max_length=150,
+        verbose_name=_("HACT Risk Rating Name"),
```
Comment (robertavram) [inline src/etools/applications/partners/models.py:512]: change this to "Highest Risk Rating Name"

File: src/etools/applications/partners/models.py (hunk 1 above)
Comment (reinbach) [reply src/etools/applications/partners/models.py]: :+1:

File: src/etools/applications/partners/models.py (hunk 2 above)
Comment (reinbach) [reply src/etools/applications/partners/models.py]: :+1:

**Pull request merged**